### DISCUSSION TITLE
What is the need for flag 'LYS_MAND_FALSE' to exist

### DISCUSSION THREAD

**Discussion by lyzliyuzhi91** [CESNET/libyang#1736] [Q&A]
https://github.com/CESNET/libyang/blob/25ede0234ab1cde50e7e7f7efde1a2f43bc80fdc/src/tree_schema.h#L897

I find nowhere to use the flag 'LYS_MAND_FALSE'  in the master branch. 
But in libyang1, it will be used to print yang/yin.

**Comment by michalvasko** [comment]
It is only hidden by the use of [`LYS_MAND_MASK`](https://github.com/CESNET/libyang/blob/25ede0234ab1cde50e7e7f7efde1a2f43bc80fdc/src/tree_schema.h#L900).
